Casamari/MNC010.xml

```diff
@@ -33,49 +36,143 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <textLang mainLang="gez"/>
+                     <note>The work comprises 56 stanzas.</note>
+                     <incipit xml:lang="gez">
+                        <locus target="#20r"/>
+                        <pb n="20r"/>
+                        <hi rend="rubric">ሰላም፡ ለዝክረ፡ ስምከ፡ ስመ፡ መሐላ፡ ዘኢይኄሱ። ዘአንበረ፡ ቅድመ፡</hi>አግዚአብሔር፡
+                        በአትሮንሱ። <hi rend="rubric">ኢየሱስ፡ ክርስቶስ፡</hi> ለዳዊት፡ ባሕርየ፡ ከርሡ። አክሊለ፡ ስምከ፡ እንዘ፡
+                        ይትቄጸል፡ ባርእሱ። አኅጉረ፡ ፀር፡ ወረሰ፡ ኢያሱ።</incipit>
+                     <explicit xml:lang="gez">
+                        <locus target="#33v"/><hi rend="rubric">አምላከ፡</hi>ምድር፡ ወሰማያት፡ አምላከ፡ ባሕር፡
+                        ወቀላያት። ወአምላከ፡ ኵሉ፡ ፍጥረት ። አምላኮሙ፡ አንት፡ ለአበው፡ ቀደምት። አምላኮሙ፡ ለመላእክት፡ አምላከ፡ ለነቢያት፡
+                        ወሐዋርያ ት። አምላከ፡ ጻድቃን፡ ወሰማዕት። መሐረነ፡ ለነ፡ አምላክነ፡ እስመ፡ ግብረ፡ እዴከ፡ ንሕነ። ወኢትዝክር፡ ኵሎ፡
+                        አበሳነ።</explicit>
                   </msItem>
+
                   <msItem xml:id="ms_i4">
-                     <locus from="35v" to="40r"/>
-                     <title type="complete" ref="LIT4963Naahabeya">‘Come to me, o my Lord Jesus Christ’</title>
+                     <locus target="#34r"/>
+                     <title type="complete" ref="LIT2993RepCh274">Hymn to God</title>
+                     <textLang mainLang="gez"/>
+                     <note>See the transcription in <ref target="#HymnGod1"/>.</note>
+                  </msItem>
+
+                  <msItem xml:id="ms_i5">
+                     <locus from="34r" to="34v"/>
+                     <title type="complete" ref="LIT6232HymnGod">Hymn to God</title>
+                     <textLang mainLang="gez"/>
+                     <note>See the transcription in <ref target="#HymnGod2"/>.</note>
+                  </msItem>
+
+                  <msItem xml:id="ms_i6">
+                     <locus from="34v" to="35r"/>
+                     <title type="complete" ref="LIT2988RepCh269">Hymn to God</title>
+                     <textLang mainLang="gez"/>
+                     <note>See the transcription in <ref target="#HymnGod3"/>.</note>
+                  </msItem>
+
+                  <msItem xml:id="ms_i7">
+                     <locus from="35r" to="35v"/>
+                     <title type="complete" ref="LIT3002RepCh282">Hymn to God</title>
+                     <textLang mainLang="gez"/>
+                     <note>See the transcription in <ref target="#HymnGod4"/>.</note>
+                  </msItem>
+
+                  <msItem xml:id="ms_i8">
+                     <locus from="35v" to="38v"/>
+                     <title type="complete" ref="LIT4963Naahabeya"/>
                      <textLang mainLang="gez"/>
+                     <incipit xml:lang="gez">
+                        <locus target="#35v"/>
+                        <hi rend="rubric">ነዓ፡ ኀቤየ፡ ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ ወልደ፡ እግዚአ</hi><del
+                           rend="erasure" cause="superfluous">እግዚአ</del>ብሔር፡ ሕያው። ወወልደ፡ <hi
```

Comment:
Wait, I don't know if this is okay. I found it reasonable, since we need a diffrentiatin if something was erased ebcause it was superflous or for other reasons (like names)